plane.c

```diff
@@ -30,6 +30,37 @@ static int guess_plane_zpos_from_type(struct liftoff_device *device,
 	return 0;
 }
 
+static void log_plane(struct liftoff_plane *plane)
```

Comment:
Hmm. Maybe we can ask users to run [drm_info](https://github.com/ascent12/drm_info) if we need this kind of information?

Reply:
I see your point though for rapid development at least short term it makes sense to keep it in. It gives a good overview of only the relevant parts of the system without the need for other tools.

It also logs the zpos value that is not always a drm property.

On the other side I have just added a commit that logs the current state of all relevant planes on apply on an output. So you get an overview there as well.

So I'm fine either way.